frontend/animal/Experiment.js

```diff
@@ -69,7 +69,10 @@ class Experiment {
             .add_tbody_tr("Vehicle", this.data.vehicle)
             .add_tbody_tr("Animal diet", this.data.diet)
             .add_tbody_tr("Guideline compliance", this.data.guideline_compliance)
-            .add_tbody_tr("Description and animal husbandry", this.data.description);
+            .add_tbody_tr(
+                "Comments (eg. description, animal husbandry, etc.)",
```

Comment:
I think we should still call this one comments, but we want the form view to have the extra help text.